### PR TITLE
contractor uniform tweaks

### PR DESCRIPTION
**Pull request by ParadoxSpace**
Brought back the old Eridani uniform, made its' sleeves rollable. The pre-existing one is now the 'alt' version. Also, makes the IRU coat ventillated.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
now puts Idris and Necro in the faction list

**Comment by ParadoxSpace** [comment]
one last thing: adjusts the hand sprites on the idris coat to not be anus, and swaps the little insignia because satchel straps block it

**Comment by ParadoxSpace** [comment]
lastly, new Idris uniform and some visual fixes w/ the IRU coat

**Comment by VTCobaltblood** [comment]
You changed the file name (iru_coat to irucoat), but didn't change the iconstate.

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -531,10 +531,14 @@
 	armor = list(melee = 50, bullet = 30, laser = 30, energy = 15, bomb = 40, bio = 0, rad = 0)
 
 /obj/item/clothing/suit/armor/vest/idris
-	name = "Idris reclamation unit coat "
+	name = "Idris Reclamation Unit coat"
 	desc = "A coat worn by the Idris reclamation units, notorious across space."
 	icon_state = "iru_coat"
```
Comment (VTCobaltblood) [inline code/modules/clothing/suits/armor.dm]: You forgot to change this when renaming the icon.

Comment (ParadoxSpace) [reply code/modules/clothing/suits/armor.dm]: handled

**Comment by ParadoxSpace** [comment]
addressed icon issues

**Pull request merged**